modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -232,6 +232,38 @@ For more information, see
 ====
 
 
+ifeval::[{suma-content} == true]
+== Install {productname} on {sles} {bci-suma}
+
+Alternatively, you can install {productname} on {sles} {bci-suma}.
+The following procedure describes the main steps of the installation process.
+
+.Procedure: Installing {productname} on {sles} {bci-suma}
+. Locate and downlaod {sles} {bci-suma} [literal]``.iso`` at https://www.suse.com/download/sles/.
+. Make sure that the {scc} code you plan to use has the Extensions enabled.
+. Start the installation of {sles} {bci-suma}.
+  .. On the [literal]``Language, keyboard and product selection`` select the product to install.
+  .. On the [literal]``License agreement`` read the agreement and check [checkbox]``I Agree to the License Terms``.
+. Select the registration method. 
+. In the screen [literal]``Extensions and Modules Selection`` check the following:
+  * {productname} Server Extension
+  * Basesystem Module
+  * Containers Module
+. Complete the installation.
+. When the installation completes, log in to the newly installed server as root.
+. As root, run
++
+----
+zypper install podman
+----
++
+. Finish the {productname} installation on {sles} {bci-suma} by running command:
++
+----
+mgradm install podman
+----
```

Comment:
@srbarrios @Bischoff do you know if the needed aardvark-dns/netvark package will be installed automatically as part of this command or do we need to mention that?

Reply:
@admd they should be installed automatically. the spec file  of uyuni-tools has:

~~~
%if 0%{?suse_version}
Requires:       (aardvark-dns if podman)
Requires:       (netavark if podman)
%endif
~~~

Reply:
than you Marina